src/BarcodeScannerForPhoneGap/widget/BarcodeScannerForPhoneGap.js

```diff
@@ -10,6 +10,7 @@ define([
         buttonLabel: "",
         buttonClass: "",
         onchangemf: "",
+        onchangeNanoFlow: "",
```

Comment:
Choose one type of casing (camel case) => onChangeNanoflow

Reply:
Done